targets/curie_bsp/include/jerry-libc-defs.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2015-2016 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef JERRY_LIBC_DEFS_H
+#define JERRY_LIBC_DEFS_H
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
+
+/**
+ * Attributes
+ */
+#define __attr_unused___   __attribute__((unused))
+#define __attr_used___     __attribute__((used))
+#define __attr_noreturn___ __attribute__((noreturn))
+#define __attr_noinline___ __attribute__((noinline))
+
+#define RAND_MAX (0x7fffffffu)
+
+void __attribute__ ((noreturn)) exit (int);
+void __attribute__ ((noreturn)) abort (void);
+int rand (void);
+void srand (unsigned int);
+
+#endif /* !DEFS_H */
```

Comment:
If it turns out that this header is needed (I'm still undecided), then the comment has to be fixed (`/* !JERRY_LIBC_DEFS_H */`)